plugins/woocommerce-admin/client/marketplace/components/header-account/header-account.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__menu-item span {
+		white-space: normal;
+	}
+
+	&__menu-item-link {
+		display: inline-flex;
+		align-items: center;
+		color: unset;
+		text-decoration: none;
```

Comment:
The linter doesn't seem to complain when we don't order rules alphabetically, like it does on WCCOM... but it would still be nice to do it, IMHO.

Reply:
Agreed, updating in ebcb029b2e0561e085c47546b8d68583ea092824